float32[4] rotation_vec
float32 yaw
int8 shape_type
int8[] remap_id

